test/test_nn.py

```diff
@@ -4843,6 +4843,63 @@ def test_embedding_functional(self):
 
         self.assertEqual(res_old, res_F)
 
+        def test_embedding_with_kwargs(self):
```

Comment:
It's not clear to me what this is testing - the logic here doesn't relate to `__torch_function__`. Imo you can just leave this test out.

Reply:
Thanks for reviewing and commenting. I also got the similar feeling as you. But since I was asked to add some tests for this PR, I tried to guess this is the place where the test may be. Or maybe you can let me know where should I put the test for this. My next PR around sharded embedding will cover this anyway.

Reply:
Here is PR I mentioned: https://github.com/pytorch/pytorch/pull/67799. Because of this one, I need to change it to kwargs. We can get it from positional args but that brings in implicit dependency. So I guess kwargs are better to go.